ui/presentation_time.js

```diff
@@ -64,8 +64,8 @@ shaka.ui.PresentationTimeTracker = class extends shaka.ui.Element {
   updateTime_() {
     const isSeeking = this.controls.isSeeking();
     let displayTime = this.controls.getDisplayTime();
-    const duration = this.video.duration;
     const seekRange = this.player.seekRange();
+    const duration = seekRange.end - seekRange.start;
```

Comment:
Why is this more correct?  I would expect this to be a no-op for VOD, and the live branch doesn't use duration at all.

Reply:
I don't know how to explain it very well, but in my empirical tests it gives better results because it does not take into account the initial gaps

Reply:
When does VOD HLS in sequence mode have initial gaps?

Reply:
Yes